modules/atomic-widgets/atomic-dynamic-tags/module.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\AtomicDynamicTags;
+
+use Elementor\Modules\AtomicWidgets\Controls\Section;
+use Elementor\Modules\AtomicWidgets\Controls\Types\Select_Control;
+use Elementor\Modules\AtomicWidgets\Controls\Types\Text_Control;
+use Elementor\Modules\AtomicWidgets\Schema\Atomic_Prop;
+use Elementor\Modules\AtomicWidgets\Schema\Constraints\Enum;
+
+class Module {
+	public function register_hooks() {
+		add_filter( 'elementor/editor/localize_settings', fn( $settings ) => $this::add_atomic_dynamic_tags_settings( $settings ) );
+	}
+
+	private function add_atomic_dynamic_tags_settings( $settings ) {
+		$settings['atomicDynamicTags'] = self::transform_dynamic_tags_to_atomic( $settings['dynamicTags']['tags'] );
```

Comment:
Do you still need it static?

```suggestion
		$settings['atomicDynamicTags'] = $this->transform_dynamic_tags_to_atomic( $settings['dynamicTags']['tags'] );
```

Reply:
I will change it to private and remove the static